cli-tools/src/main/java/com/radixdlt/cloud/AWSSecrets.java

```diff
@@ -260,6 +270,7 @@ private static void generateAndStoreKey(
             publicKeyFileAwsSecret, publicKeyFileSecretName, awsSecretsOutputOptions, false, false);
       } catch (Exception e) {
         System.out.println(e);
+        throw new RuntimeException(e);
```

Comment:
(nitpick)

If you re-throw, please skip the log.
(in 99% cases it only leads to a duplicated log)

Reply:
Agreed, thanks. Good spot